ci/after-build-check-tasks.js

```diff
@@ -1,11 +1,12 @@
 const util = require('./ci-util');
 const fs = require('fs');
 const path = require('path');
-var crypto = require('crypto');
+const semver = require('semver');
```

Comment:
What is the version of `semver` package is used?

Reply:
The ci scripts share semver package with the main build system, so [it's 4.3.3](https://github.com/microsoft/azure-pipelines-tasks/blob/master/package.json#L45)